### DISCUSSION TITLE
Is there a way to avoid waiting for the full load of the page

### DISCUSSION THREAD

**Discussion by oltarasenko** [keathley/wallaby#596] [Q&A]
Hey, I just wonder if it's somehow possible to avoid the full load of the page, before Wallby.visit will return anything?

**Comment by mhanberg** [comment]
No. You're mostly limited to what a real browser can do.

**Comment by michallepicki** [comment]
You could try navigating by executing a script asynchronously: https://hexdocs.pm/wallaby/Wallaby.Browser.html#execute_script_async/2
